docs/workflow-design.md

```diff
@@ -1,5 +1,7 @@
 # How Katib v1beta1 tunes hyperparameters automatically in a Kubernetes native way
 
+TODO (andreyvelich): This doc is out of date. We should update it and move to the Kubeflow website.
```

Comment:
Once we open the issue to update this doc, any contribution may take the issue.

Reply:
Yes, let me create an issue.